synapse/tests/test_lib_revision.py

```diff
@@ -0,0 +1,37 @@
+from synapse.tests.common import *
+
+import synapse.lib.revision as s_revision
+
+class TestRevr(s_revision.Revisioner):
```

Comment:
I get a bunch of warnings: `cannot collect test class 'TestRevr' because it has a __init__ constructor` when I run the tests.

Reply:
``TestRevr`` needs to be renamed ``TstRevr`` so that pytest doesn't pick up on it